www/artists/delete.php

```diff
@@ -0,0 +1,82 @@
+<?
+use function Safe\session_unset;
+
+$artist = null;
+
+try{
+	if(Session::$User === null){
+		throw new Exceptions\LoginRequiredException();
+	}
+
+	if(!Session::$User->Benefits->CanReviewOwnArtwork){
+		throw new Exceptions\InvalidPermissionsException();
+	}
+
+	session_start();
+
+	$exception = HttpInput::SessionObject('exception', Exceptions\AppException::class);
+
+	$artist = Artist::GetByUrlName(HttpInput::Str(GET, 'artist-url-name') ?? '');
+	$artworkCount = count(Artwork::GetAllByArtist($artist->UrlName, Enums\ArtworkFilterType::Admin, null /* submitterUserId */));
+
+	if($exception){
+		http_response_code(Enums\HttpCode::UnprocessableContent->value);
+		session_unset();
+	}
+}
+catch(Exceptions\ArtistNotFoundException){
+	Template::ExitWithCode(Enums\HttpCode::NotFound);
+}
+catch(Exceptions\LoginRequiredException){
+	Template::RedirectToLogin();
+}
+catch(Exceptions\InvalidPermissionsException){
+	Template::ExitWithCode(Enums\HttpCode::Forbidden);
+}
+?><?= Template::Header(title: 'Delete ' . $artist->Name, css: ['/css/artwork.css']) ?>
+<main>
+	<section class="narrow">
+		<nav class="breadcrumbs">
+			<a href="<?= $artist->Url ?>"><?= $artist->Name ?></a> →
+		</nav>
+		<h1>Delete</h1>
+
+		<?= Template::Error(exception: $exception) ?>
+
+		<form method="<?= Enums\HttpMethod::Post->value ?>" action="/artists/<?= $artist->UrlName ?>">
+			<input type="hidden" name="_method" value="<?= Enums\HttpMethod::Delete->value ?>" />
+			<p>Are you sure you want to permanently delete <?= Formatter::EscapeHtml($artist->Name) ?>?</p>
+			<label class="icon user">
+				<span>Canonical Artist</span>
+				<span>Reassign artwork by <?= Formatter::EscapeHtml($artist->Name) ?> to this artist. Total artworks to reassign: <?= $artworkCount ?></span>
+				<datalist id="artist-names-except-this-artist">
+					<? foreach(Artist::GetAll() as $a){ ?>
+						<? if($a->ArtistId == $artist->ArtistId){ continue; } ?>
```

Comment:
Instead of `continue`, test if `$a->ArtistId != $artist->ArtistId` so it's a more natural brace structure instead of a short circuit

Reply:
Done